### ISSUE TITLE
Code release 

### ISSUE THREAD

**Issue by FBehrad** [SysCV/sam-hq#2]
Hello, 
Thank you for this excellent job. 
Do you have any plans to release the code and weights?

**Comment by lkeab** [comment]
Yes. We will release it soon after a good preparation/organization. Stay tuned!

**Comment by zzcgithub** [comment]
respect

**Comment by MohamedAliRashad** [comment]
@lkeab Consider a huggingface demo also
